src/main/java/uk/ac/ebi/ena/readtools/fastq/ena/Fastq2Sam.java

```diff
@@ -121,16 +123,33 @@ public void create( Params p ) throws IOException {
                 }
             }
         } while( again );
-       
+
         for( AutoNormalizerDataSpotProducer producer : producers ) {
             if( !producer.isOk() ) {
                 throw new RuntimeException(producer.getStoredException());
             }
+
+            totalRecordCount += producer.getRecordCount();
+            totalBaseCount += producer.getTotalBaseCount();

```

Comment:
Could we please discuss this part? I think we have one producer for single Fastqs and two producers for paired Fastqs. In the end we will create a single BAM file. Logically we should count the number of reads and bases when we have constructed FastqSpot and are committed to storing it in the BAM file. I would like to understand and double check what is happening here as we are summing together read and base counts from two producers. I would expect to see the read and base counts coming from a single source ('owner' of FastqSpots).